src/win32/pthread/pthread_mutex_destroy.c

```diff
@@ -37,7 +37,9 @@
 #include "pthread.h"
 #include "implement.h"
 
-int getpid(void);
+#if REALM_UWP
+int _getpid(void);
+#endif
```

Comment:
Have you checked if an updated version - with UWP support - of this 3rd party library exists? I think we would prefer to keep 3rd party code unmodified to the extend possible.